File: day10a.java
public class day10a {
    public static void main(String[] args) {
        int n = 5;
        int[] arr = { 1, 2, 3, 4, 5 };

        printArrayElements(arr, n);
    }

    public static void printArrayElements(int[] arr, int n) {
        for (int i = 0; i < n; i++) {
            System.out.print(arr[i] + " ");
        }
    }
}

File: day10a.cpp
#include <iostream>
using namespace std;

void printArrayElements(int arr[], int n)
{

    for (int i = 0; i < n; i++)
    {
        cout << arr[i] << " ";
    }
}

int main()
{
    int n = 5;
    int arr[] = {1, 2, 3, 4, 4};

    printArrayElements(arr, n);

    return 0;
}